vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -155,23 +155,30 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
         return html.Div([dashboard_title, settings, page_title, nav_bar, nav_panel, control_panel, components])
 
     def _arrange_page_divs(self, page_divs: _PageDivsType):
```

Comment:
Is `_PageDivsType` still correct or does it need updating? (Hopefully mypy would have caught if it needed updating, but good to check...)